app/components/reactions/reactions.js

```diff
@@ -55,9 +60,24 @@ export default class Reactions extends PureComponent {
     }
 
     render() {
+        const {reactions}= this.props;
+        const styles = getStyleSheet(this.props.theme);
+        let addMoreReactions = null;
+        if (reactions.size != 0) {
```

Comment:
perhaps is best if you do the following

```js
if (!reactions.size) {
    return null;
}

const addMoreReactions = (
    <TouchableOpacity
         onPress={this.props.onAddReaction}
        style={[styles.reaction]}
    >
       <Text style={styles.more}>{'+'}</Text>
   </TouchableOpacity>
);
```

and then leave the return as it is.

so if no reactions it just return and if you have at least one reaction then it shows you the `+` option

Reply:
done

thanks so much for your review and support @enahum